DataImport.Web/Infrastructure/MinimumRequiredSetupValidationFilter.cs

```diff
@@ -31,25 +27,14 @@ public MinimumRequiredSetupValidationFilter(DataImportDbContext dbContext)
 
         public override void OnActionExecuting(ActionExecutingContext filterContext)
         {
-            if (filterContext.Controller is ConfigurationController ||
-                filterContext.Controller is RecoverUserController ||
+            if (filterContext.Controller is RecoverUserController ||
                 filterContext.Controller is ApiServersController ||
                 filterContext.Controller is OpenIdConnectController ||
                 (filterContext.Controller is HomeController && (string) filterContext.RouteData.Values["action"] == "UserUnauthorized"))
             {
                 return;
             }
 
-            if (!_dbContext.ApiServers.Any())
-            {
-                filterContext.Result = new RedirectToRouteResult(
-                    new RouteValueDictionary
-                    {
-                            { "controller", "Configuration" },
-                            { "action", "Index" },
-                            { nameof(EditConfiguration.Query.OdsApiServerException), true }
-                    });
-            }
         }
```

Comment:
@lchavarriaga 
Please consider to verify the flow - if no api server connections defined, then the page needs to be redirected to API-connections page.

Reply:
Fixed